persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/EnsemblRepositoryImpl.java

```diff
@@ -20,6 +25,53 @@ public EnsemblRepositoryImpl(MongoTemplate mongoTemplate)
 
     public static final String CANONICAL_TRANSCRIPTS_COLLECTION = "ensembl.canonical_transcript_per_hgnc";
     public static final String TRANSCRIPTS_COLLECTION = "ensembl.biomart_transcripts";
+    public static final String HGNC_COLLECTION = "hgnc";
+
+    @Override
+    public EnsemblGene getCanonicalEnsemblGeneIdByHugoSymbol(String hugoSymbol) {
+        BasicDBList or = new BasicDBList();
```

Comment:
Using a `Query` instance (if possible) seems easier to read to me: https://github.com/genome-nexus/genome-nexus/blob/b331bceaa522f6b543f25b3685d0494ad3b5ae9b/persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/PdbHeaderRepositoryImpl.java#L32